src/frontends/tensorflow/src/op/parallel_dynamic_stitch.cpp

```diff
@@ -0,0 +1,83 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "op_table.hpp"
+#include "openvino/opsets/opset9.hpp"
+
+using namespace std;
+using namespace ov::opset9;
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+namespace op {
+
+OutputVector translate_parallel_dynamic_stitch_op(const NodeContext& node) {
+    // format for inputs: [indices1, indices2, ..., indicesN, data1, data2, ..., dataN]
+    // so we expect at least 2 input and the total number of inputs must be divisible by 2
+    default_op_checks(node, 2, {"ParallelDynamicStitch", "DynamicStitch"});
+    auto in_size = node.get_input_size();
+    TENSORFLOW_OP_VALIDATION(node,
+                             in_size % 2 == 0,
+                             "The total number of inputs to DynamicStitch operation "
+                             "must be divisible by 2.");
+
+    size_t N = in_size / 2;
+    OutputVector indices_to_concat;
+    OutputVector data_to_concat;
+    auto data_element_type = node.get_input(N).get_element_type();
+    auto const_minus_one = std::make_shared<Constant>(ov::element::i32, Shape{1}, -1);
+    auto const_zero = std::make_shared<Constant>(ov::element::i32, Shape{1}, 0);
+    auto const_one = std::make_shared<Constant>(ov::element::i32, Shape{1}, 1);
+    for (size_t i = 0; i < N; ++i) {
+        auto indices = node.get_input(static_cast<int>(i));
```

Comment:
Wouldn't it be easier if you had `i` of type `int` instead of `size_t`. You use it only in `get_input` function.

Reply:
done